configure.py

```diff
@@ -534,6 +534,7 @@ def has_re2c() -> bool:
              'build_log',
              'clean',
              'clparser',
+             'clsourcedependencies_parser',
```

Comment:
There is no definition for `NINJA_ENABLE_CL_SOURCE_DEPENDENCIES` in this file, which means that these will always be compiled as empty, which is slightly confusing.

Either do not add them, and add a comment explaining why, or set the macro, or only add these for the Win32 case.

Reply:
I added `--rapidjson-root-dir` and `RAPIDJSON_ROOT_DIR` to `configure.py`, along with help and README content.